### ISSUE TITLE
Making results consistent when the year is shifted

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#4888]
It is now easy to analyse data with the shifted day of year and the ordinary year column.  I made this mistake for the start of the rains in Dodoma and got plausible, but wrong, answers.  

We can't think why this would ever be sensible to do.  Can we warn and make it difficult to avoid having the   shifted year when we do the shifted day?

**Comment by dannyparsons** [comment]
Yes we can store the shift start day in the metadata and then check that they match in the dialogs.
